lib/health_check/middleware_health_check.rb

```diff
@@ -0,0 +1,42 @@
+module HealthCheck
+  class MiddlewareHealthcheck
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      uri = env['PATH_INFO'.freeze]
+      if uri.include? HealthCheck.uri
+        response_type = uri.sub! '/' + HealthCheck.uri + '.' , ''
+        response_method = 'response_' + response_type.to_s
```

Comment:
response_method = 'response_' + (response_type || 'default').to_s
then you can rename `response_` to `response_default`